client/descriptions.go

```diff
@@ -204,6 +204,7 @@ var FieldKindStringToEnumMapping = map[string]FieldKind{
 	"String":     FieldKind_STRING,
 	"[String]":   FieldKind_NILLABLE_STRING_ARRAY,
 	"[String!]":  FieldKind_STRING_ARRAY,
+	"Bytes":      FieldKind_BYTES,
```

Comment:
todo: I think this is a strange way of representing this, and that `[Byte]` is more sensible and fits much better with the rest of the types, and any potential future `Byte`, `[Byte!]` or `Byte!` types.

Reply:
Updated to `Blob` as we discussed in the standup.